src/Core/Framework/Routing/ApiRouteScope.php

```diff
@@ -12,9 +13,9 @@
 #[Package('framework')]
 class ApiRouteScope extends AbstractRouteScope implements ApiContextRouteScopeDependant
 {
-    final public const ID = 'api';
+    final public const ID = DefinitionService::API;
 
-    protected array $allowedPaths = ['api', 'sw-domain-hash.html'];
+    protected array $allowedPaths = [self::ID, 'sw-domain-hash.html'];
```

Comment:
also here, it is convention that the id is the same as the path prefix, however that's not given, therefore I would rather hard code the `api` prefix here again